nipy/algorithms/statistics/formula/formulae.py

```diff
@@ -526,7 +527,7 @@ def fromrec(rec, keep=[], drop=[]):
         if keep:
             return np.sum([t for n, t in f.items() if n in keep])
         else:
-            return np.sum(f.values())
+            return np.sum(list(f.values()))
```

Comment:
> In [1]: a = {'a': 1, 'b': 2}

> In [2]: import numpy as np

> In [3]: np.sum(a.values())
> Out[3]: dict_values([2, 1])

> The list values are an iterator, and we need something that gets converted to
> an array.

ah... which python/numpy was that? with 2.7.10

    In [6]: a = {'a': 1, 'b': 2}

    In [7]: np.sum(a.values())
    Out[7]: 3

    In [8]: np.__version__
    Out[8]: '1.8.2'

that is why I have assumed that it just works and no explicit 'list'ing is
needed.  gotcha now